apps/timetracking/app/src/components/WeekTable/WeekTable.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { WeekDay } from './WeekDay'
+const WeekTable = (props) => {
+  var filteredWorkedHours = props.workedHours
+  var weekDayHours = [0,0,0,0,0,0,0]
+  var weekDayTokens = [0,0,0,0,0,0,0]
+  
+  for(let i=0; i< filteredWorkedHours.length; i++) {
```

Comment:
Do we need to calculate this everytime for every week or can we save it in variables?